src/ducks/apps/components/ApplicationPage/Details.jsx

```diff
@@ -12,6 +12,7 @@ import Maintenance from './Maintenance'
 import ReactMarkdownWrapper from 'ducks/components/ReactMarkdownWrapper'
 import { getContext, REGISTRY_CHANNELS } from 'ducks/apps'
 import getChannel from 'lib/getChannelFromSource'
+import { getTranslatedManifestProperty } from 'lib/helpers'
```

Comment:
Haven't we a extendI18n mechanism in the store ?

Reply:
This helper is to get the localised info here, not related to the transifex extension if I understand correctly here